### PR TITLE
Start docs

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

File: docs/deployment.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+#. Go to `register your own Firefox Accounts OAuth Client`_:
+
+    * Client name: moz-dev-dash-username
+    * Redirect URI: https://moz-dev-dash-username.herokuapp.com/accounts/fxa/login/callback/
+    * Trusted Mozilla Client: **CHECKED**
+
+   Be sure to copy the client secret - you can't see it again.
+
+#. Go to https://moz-dev-dash-username.herokuapp.com/admin/socialaccount/socialapp/add/
+   to `enable Firefox Accounts Auth`_ like a local machine; this time using your own new Firefox Accounts OAuth Client ID and Secret
```
Comment (openjck) [inline docs/deployment.rst]: ERROR: Unknown target name: "enable firefox accounts auth".

File: docs/index.rst [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+
+Resources
+---------
+.. image:: https://travis-ci.org/groovecoder/push-dev-dashboard.png?branch=master
+   :target: https://travis-ci.org/groovecoder/push-dev-dashboard
+   :alt: Travis-CI Build Status
+.. image:: https://coveralls.io/repos/groovecoder/push-dev-dashboard/badge.png
+    :target: https://coveralls.io/r/groovecoder/push-dev-dashboard 
+.. image:: https://requires.io/github/groovecoder/push-dev-dashboard/requirements.png?branch=master
```
Comment (openjck) [inline docs/index.rst:27]: I'm getting a 404 from [https://requires.io/github/groovecoder/push-dev-dashboard/requirements.png?branch=master](https://requires.io/github/groovecoder/push-dev-dashboard/requirements.png?branch=master). Is that a known issue for the time being?

File: docs/index.rst [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+   :target: https://requires.io/github/groovecoder/push-dev-dashboard/requirements/?branch=master
+   :alt: Requirements Status
+
+:Code:          https://github.com/groovecoder/push-dev-dashboard
+:License:       AGPLv3; see `LICENSE file
+                <https://github.com/groovecoder/push-dev-dashboard/blob/master/LICENSE>`_
```
Comment (openjck) [inline docs/index.rst]: Looks like the LICENSE file hasn't been added yet

File: docs/development.rst [hunk 4]
```diff
@@ -0,0 +1,119 @@
+Development
+===========
+
+Requirements
+------------
```
Comment (openjck) [inline docs/development.rst:5]: Also virtualenv

File: docs/development.rst [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+.. _Install requirements: http://pip.readthedocs.org/en/latest/user_guide.html#requirements-files
+.. _Create a superuser: https://docs.djangoproject.com/en/1.7/ref/django-admin/#django-admin-createsuperuser
+
+
+Run it
+------
+
+#. Source the ``.env`` file to set environment config vars (Can also use `autoenv`_)::
+
+    source .env
```
Comment (openjck) [inline docs/development.rst]: I notice this is also done earlier in the steps (line 31 currently). Does it need to be done again?

Comment (openjck) [inline docs/development.rst]: Nevermind, I understand now.

File: docs/development.rst [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+
+.. _Add a django-allauth social app: http://127.0.0.1:8000/admin/socialaccount/socialapp/add/
+.. _Log out of the admin account: http://127.0.0.1:8000/admin/logout/
+
+
+Run the Tests
+-------------
+Running the test suite is easy::
+
+    ./manage.py test -s --noinput --logging-clear-handlers
```
Comment (openjck) [inline docs/development.rst]: I'm getting this, even with virtualenv active:

`./manage.py: Permission denied`

File: docs/deployment.rst [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+#. Sign in at https://moz-dev-dash-username.herokuapp.com/ with a Firefox
```
Comment (openjck) [inline docs/deployment.rst:48]: I get a 404 after signing in, but it looks like that's a known issue atm?

**Comment by openjck** [comment]
Otherwise looks great!

File: docs/development.rst (hunk 6 above)
Comment (openjck) [inline docs/development.rst]: Permissions are -rw-r--r-- and I'm the owner of the file, so it looks fine...

File: docs/deployment.rst (hunk 7 above)
Comment (groovecoder) [inline docs/deployment.rst:48]: Yup, that's https://github.com/groovecoder/push-dev-dashboard/issues/2.

File: docs/index.rst (hunk 2 above)
Comment (groovecoder) [inline docs/index.rst:27]: Known bug for now. Filed issue here: https://github.com/groovecoder/push-dev-dashboard/issues/8

File: docs/index.rst (hunk 3 above)
Comment (groovecoder) [inline docs/index.rst]: oops, wrong license too. I'll just call it MPL 2

File: docs/development.rst (hunk 4 above)
Comment (groovecoder) [inline docs/development.rst:5]: Added.

**Comment by groovecoder** [comment]
Merging my own PR like it ain't no thang.

**Pull request merged**